examples/gallery/grid/grdview_surface.py

```diff
@@ -0,0 +1,50 @@
+"""
+Plotting a surface
+------------------
+
+The :meth:`pygmt.Figure.grdview()` method can plot 3-D surfaces with ``surftype="s"``. Here,
+we supply the data as an :class:`xarray.DataArray` with the coordinate vectors ``x`` and
+``y`` defined. Note that the ``perspective`` argument here controls the azimuth and
+elevation angle of the view. We provide a list of two arguments to ``frame`` — the
+second argument, prepended with ``"z"``, specifies the :math:`z`-axis frame attributes.
+Specifying the same scale for the ``projection`` and ``zcale`` arguments ensures equal
+axis scaling.
+"""
+
+import pygmt
+import numpy as np
+import xarray
```

Comment:
```suggestion
import xarray as xr
```
This is the most commonly used way to import the xarray package.